From 5dda81412db1609f67035957ab65a6c726228e00 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Tue, 14 Jul 2015 11:12:56 -0700
Subject: [PATCH 20/20] Ticket #48194 - CI test: fixing test cases for ticket
 48194

Description: nsSSL3Ciphers preference not enforced server side
. Test Case 6 - wrong expectation for RC4-SHA
. Test Case 7 - removing a extra space in nsSSL3Ciphers

(cherry picked from commit f69ce333052f7f33350fd4038b8f598f650a743f)
(cherry picked from commit ca9e6f9292289a7b6b7f57602555bf9aeb9ba9de)
---
 dirsrvtests/tickets/ticket48194_test.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/dirsrvtests/tickets/ticket48194_test.py b/dirsrvtests/tickets/ticket48194_test.py
index 18739ca..17e179a 100644
--- a/dirsrvtests/tickets/ticket48194_test.py
+++ b/dirsrvtests/tickets/ticket48194_test.py
@@ -295,7 +295,7 @@ def test_ticket48194_run_4(topology):
     Default ciphers are enabled.
     default allowWeakCipher
     """
-    _header(topology, 'Test Case 5 - Check no nsSSL3Ciphers (default setting) with default allowWeakCipher')
+    _header(topology, 'Test Case 5 - Check no nsSSL3Ciphers (-all) with default allowWeakCipher')
 
     topology.standalone.simple_bind_s(DN_DM, PASSWORD)
     topology.standalone.modify_s(ENCRYPTION_DN, [(ldap.MOD_DELETE, 'nsSSL3Ciphers', '-all')])
@@ -326,7 +326,7 @@ def test_ticket48194_run_5(topology):
     os.system('touch %s' % (topology.standalone.errlog))
     topology.standalone.start(timeout=120)
 
-    connectWithOpenssl(topology, 'RC4-SHA', True)
+    connectWithOpenssl(topology, 'RC4-SHA', False)
     connectWithOpenssl(topology, 'AES256-SHA256', True)
 
 def test_ticket48194_run_6(topology):
@@ -338,7 +338,7 @@ def test_ticket48194_run_6(topology):
     _header(topology, 'Test Case 7 - Check nsSSL3Ciphers: +all,-TLS_RSA_WITH_AES_256_CBC_SHA256  with default allowWeakCipher')
 
     topology.standalone.simple_bind_s(DN_DM, PASSWORD)
-    topology.standalone.modify_s(ENCRYPTION_DN, [(ldap.MOD_REPLACE, 'nsSSL3Ciphers', '+all,-TLS_RSA_WITH_AES_256_CBC_SHA256 ')])
+    topology.standalone.modify_s(ENCRYPTION_DN, [(ldap.MOD_REPLACE, 'nsSSL3Ciphers', '+all,-TLS_RSA_WITH_AES_256_CBC_SHA256')])
 
     log.info("\n######################### Restarting the server ######################\n")
     topology.standalone.stop(timeout=10)
-- 
1.9.3

